src/PKSim.Infrastructure/ORM/FlatObjects/FlatPopulation.cs

```diff
@@ -6,6 +6,7 @@ public class FlatPopulation : FlatObject
         public string Species { get; set; }
         public bool IsAgeDependent { get; set; }
         public bool IsHeightDependent { get; set; }
+        public bool IsBodySurfaceAreaDependent { get; set; }
```

Comment:
I'm not sure, if we really need `IsBodySurfaceAreaDependent` here.
If a species is height dependent, it is automatically BSA dependent.

We could keep the property `IsBodySurfaceAreaDependent` in `SpeciesPopulation` (for the clarity of code), but it would just return the value of `IsHeightDependent`

What do u think?